f5_os_test/polling_clients.py

```diff
@@ -71,6 +71,20 @@ def __getattr__(self, name):
         if hasattr(self.client, name):
             return getattr(self.client, name)
 
+    def _poll_call_with_exceptions(self, exceptional, call, *args, **kwargs):
+        attempts = 0
+        while attempts < self.max_attempts:
```

Comment:
Either line 76 or 82 need an =, otherwise this method will return w/out setting a retval or raising.